cala-ledger/src/tx_template/cel_context.rs

```diff
@@ -0,0 +1,11 @@
+use cala_cel_interpreter::CelContext;
+
+pub(super) fn initialize() -> CelContext {
```

Comment:
I don't like the location of this - its okay for now (as its the same as in sqlx-ledger) but the interpreter will need a bit of refactoring anyhow.